components/com_fields/layouts/fields/render.php

```diff
@@ -69,7 +85,47 @@
 		continue;
 	}
 
-	$output[] = '<li class="field-entry ' . $class . '">' . $content . '</li>';
+	if (!array_key_exists($field->group_id, $groupFields))
+	{
+		$groupFields[$field->group_id] = array();
+		if (Factory::getLanguage()->hasKey($field->group_title)) {
+			$groupTitles[$field->group_id] = Text::_($field->group_title);
+		} 
```

Comment:
```suggestion
		}
```
Remove space at end of line to fix code style error in Drone.